linux/tools.Dockerfile

```diff
@@ -16,9 +16,9 @@ RUN wget -nv https://azurecliprod.blob.core.windows.net/cloudshell-release/azure
     && dpkg -i azure-cli-latest-buster.deb \
     && rm -f azure-cli-latest-buster.deb
 
-# latest SSHArc 0.2.0 from ClI package.
+# Install latest SSHArc 0.2.0 from ClI package.
 RUN wget -nv -q http://aka.ms/SSHArc/CLI/Download \
-    && /opt/ansible/bin/python -m pip install -r ssh-0.2.0-py3-none-any.whl \
+    && python -m pip install -r ssh-0.2.0-py3-none-any.whl \
```

Comment:
The wget line will create a local file called Download, but then you try to install a file called ssh-0.2.0.py3-none-any.whl.

Change this to 
```
wget -nv -q http://aka.ms/SSHArc/CLI/Download -O ssh-0.2.0-py3-none-any.whl \
&& az extension add --system --source  ssh-0.2.0-py3-none-any.whl \
&& rm ssh-0.2.0-py3-none-any.whl
```

And remove the separate call to `RUN az extension add --system --name ssh -y`

Reply:
We download it:
`RUN az extension add --system --yes --source https://ssharc.blob.core.windows.net/ssharc/ssh-0.2.0-py3-none-any.whl`

Cli Version:
![image](https://user-images.githubusercontent.com/64445948/140975674-c12cfdc6-bfad-4d03-8037-b1250a5cbc93.png)